modules/project/README.md

```diff
@@ -1360,12 +1360,50 @@ module "bucket" {
 # tftest inventory=data.yaml e2e
 ```
 
+## API Alerts
+
+There are events within Google Cloud that should be monitored and alerted on to ensure that you are aware of any potential security issues.
+These actions are typically seen in cases of security breaches, or potential security breaches, although they can be genuine actions that are not security related, but are still important to monitor.
+
+These events are typically
+
+- Owner Role Assignment/Changes
+- Audit Logging Configuration Changes
+- Custom Roles Creation/editing/deletion
+- VPC Network Firewall Rule Changes
+- VPC Network Route Changes
+- VPC Network Changes
+- Cloud Storage IAM Permission Changes
+- SQL Instances Configuration Changes
+
+Although you may not use the services listed above, such as SQL, it is still important to monitor these events for compliance purposes
+
+To enable these alerts by default on all projects created, it is recommended to default the variable `enable_default_api_alerts` within `variables.tf` to true, 
+You will also need to set the `default_api_alerts_email` variable to the email address that will receive these alerts
+
+You can alternatively enable these alerts on a per-project basis by setting the variable `enable_api_alerts` to true on the module, along with the `default_api_alerts_email` variable
+
+```terraform
+module "project" {
+  source          = "./fabric/modules/project"
+  billing_account = var.billing_account_id
+  name            = "project"
+  parent          = var.folder_id
+  prefix          = var.prefix
+  services = [
+    "stackdriver.googleapis.com"
+  ]
+  enable_api_alerts = true
```

Comment:
can we combine both into a single `api_alerts` variable, with `.enabled` and `default_email` attributes?